api/src/routes/booking.ts

```diff
@@ -0,0 +1,22 @@
+import { Request, Response, Express } from "express";
+import { Database } from "sqlite";
+import sqlite3 from "sqlite3";
+
+export function patchBooking(
+  app: Express,
+  db: Database<sqlite3.Database, sqlite3.Statement>
+) {
+  app.patch("/api/book", async (req: Request, res: Response) => {
+    console.log(req.body);
```

Comment:
This is a PR to production, so either remove the log statement or add a TODO with an issue number